src/searchModal.js

```diff
@@ -238,7 +337,10 @@ export default function searchModalFactory(config) {
             action: 'update',
             dataset,
             context: context.shownStructure,
-            query: searchInput.val()
+            criterias: {
+                search: $searchInput.val(),
+                class: _.map(resourceSelector.getSelection(), 'uri')[0]
```

Comment:
Is this the same as `resourceSelector.getSelection()[0].uri `?

Reply:
It is not the same, as `resourceSelector.getSelection()` returns:

```
{
  "http://www.tao.lu/Ontologies/TAOItem.rdf#Item": {
    "label": "Item",
    "type": "class",
    "uri": "http://www.tao.lu/Ontologies/TAOItem.rdf#Item",
    "classUri": "http://www.tao.lu/Ontologies/TAOItem.rdf#Item",
    "signature": "c875031e47982028c84ac8bd0cd62f7c55fd2e3504555535707728176c490540",
    "state": "open",
    "selectable": true
  }
}
```